src/main/java/com/gregtechceu/gtceu/integration/emi/GTEMIPlugin.java

```diff
@@ -32,7 +33,7 @@ public class GTEMIPlugin implements EmiPlugin {
     @Override
     public void register(EmiRegistry registry) {
         registry.addCategory(MultiblockInfoEmiCategory.CATEGORY);
-        registry.addCategory(GTOreProcessingEmiCategory.CATEGORY);
+        if (!ConfigHolder.INSTANCE.compat.hideOreProcessingDiagrams) registry.addCategory(GTOreProcessingEmiCategory.CATEGORY);
```

Comment:
Just a minor nitpick, but please add a newline after the conditions in these checks:

```suggestion
        if (!ConfigHolder.INSTANCE.compat.hideOreProcessingDiagrams)
            registry.addCategory(GTOreProcessingEmiCategory.CATEGORY);
```